requirements.txt

```diff
@@ -1,3 +1,3 @@
-awscli>=1.10.16
+awscli>=1.10.48
```

Comment:
Where is this file used? I'd expect us to have to update one of our containers if we need a newer version of this.